lib/internal/util.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const prefix = '(node) ';
+const prefix = `(node:${process.pid}) `;
```

Comment:
I think using `process.title` or `process.release.name` loses value, especially when used with `cluster`. I think we should stick with `process.pid`